### PR TITLE
Auth page

### PR DESCRIPTION
**Pull request by Muzammil-khan710**
Added login and signup pages

### CODE REVIEW

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

File: src/Components/Login/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import "./Login.css"
+
+const LoginPage = () => {
+  return (
+    <div>
+        <h1>Hello, Welcome again</h1>
+      <div className="login-container">
+        <div className="login-content">
+          <h2>Log in</h2>
+          <p>Enter your email</p>
```
Comment (Ranjeetk489) [inline src/Components/Login/Login.jsx:13]: use form tag and display a button for user to submit the credentials

File: src/Components/Signup/Signup.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  return (
+    <div>
+        <h1>Signup to continue  </h1>
+        <div className="signup-container">
+        <div className="login-content">
+          <h2>Sign up</h2>
+          <p>Enter your name</p>
+          <input className="input" placeholder="Name" />  
+          <p>Enter your email</p>
+          <input className="input" type="text" placeholder="name@mail.com"/>
```
Comment (Ranjeetk489) [inline src/Components/Signup/Signup.jsx:16]: use form tag and display a button

**Review by Ranjeetk489 — APPROVED** [review]
After making the said changes code will be good to merge

**Review by Shubhamkr2610 — APPROVED** [review]

**Review by Shubhamkr2610 — APPROVED** [review]

File: src/App.js [hunk 3]
```diff
@@ -2,6 +2,8 @@ import "./App.css";
 import { Footer, Navbar, Homepage } from "./Components/Allcomps";
 import { ProductsPage } from "./Components/Products/ProductsPage";
```
Comment (Shubhamkr2610) [inline src/App.js:3]: Appreciation: good way of importing files , You are good to go . You can merge it

**Pull request merged**